addon/mixins/route.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 
 const { Mixin, RSVP, getOwner } = Ember;
+const PREFETCH_NOT_DEFINED = Object.freeze({});
```

Comment:
Either `Ember.Symbol` or `Ember.NullObject` (I don't know what either one is actually called. :smile:)